src/ResourceManager/Network/Commands.Network/help/AzureRM.Network.md

```diff
@@ -165,7 +165,10 @@ Gets the WAF configuration of an application gateway.
 Gets an application security group.
 
 ### [Get-AzureRmBgpServiceCommunity](Get-AzureRmBgpServiceCommunity.md)
-{{Manually Enter Get-AzureRmBgpServiceCommunity Description Here}}
```

Comment:
@murilogr would you mind adding a description for this cmdlet since it was removed? (you can just use the description of the cmdlet if it has one)

Reply:
Done